src/main/webapp/app/shared/user-settings/vcs-access-tokens-settings/vcs-access-tokens-settings.component.ts

```diff
@@ -0,0 +1,99 @@
+import { Component, OnInit } from '@angular/core';
+import { User } from 'app/core/user/user.model';
+import { AccountService } from 'app/core/auth/account.service';
+import { Subject, Subscription, tap } from 'rxjs';
+import { ProfileService } from 'app/shared/layouts/profiles/profile.service';
+import { PROFILE_LOCALVC } from 'app/app.constants';
+import { faCopy, faEdit, faSave, faTrash } from '@fortawesome/free-solid-svg-icons';
+import { ButtonSize, ButtonType } from 'app/shared/components/button.component';
+import { AlertService } from 'app/core/util/alert.service';
+
+@Component({
```

Comment:
We want to make our new components standalone

Reply:
Would that make sense here, as it would break the symmetry to the other settings pages? 
If it makes sense maybe it would be a better idea to refactor the whole settings module (account-settings, science....) to standalone components, in a separate PR

Reply:
Refactoring all of them soon also sounds good to me. But we should not keep adding non-standalone components, just because others are also not standalone